portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase

```diff
@@ -180,6 +180,42 @@ definition {
 		}
 	}
 
+	@description = "This test validates Fjord Theme."
+	@priority = "5"
+	test ValidateFjordTheme {
+		property portal.acceptance = "true";
+		property test.name.skip.portal.instance = "UIInfrastructureUseCase#ValidateFjordTheme";
+
+		ProductMenu.gotoPortlet(
+			category = "Sites",
+			panel = "Control Panel",
+			portlet = "Sites");
+
+		Site.addSiteTemplateCP(
+			siteName = "Fjord Name",
+			siteTemplateName = "Fjord");
+
+		Navigator.openSiteURL(siteName = "Fjord Name");
+
+		AssertClick(
+			key_pageName = "Home",
+			locator1 = "Home#PAGE",
+			value1 = "Home");
+
+		ProductMenuHelper.closeProductMenu();
+
+		SikuliAssertElementPresent(locator1 = "MediaGallery#SCREENSHOTS_FJORD_THEME_PNG");
+
+		takeScreenshot();
+
+		ProductMenu.gotoPortlet(
+			category = "Sites",
+			panel = "Control Panel",
+			portlet = "Sites");
+
+		Site.deleteCP(siteName = "Fjord Name");
```

Comment:
not needed if you're trying to clean up the test. Clean up is taken care of by tearDown. If we're testing deletion of Fjord Theme then we should split that to another test.